### PR TITLE
test: Brazilian Portuguese locale test

### PR DESCRIPTION
**Pull request by Gundersanne**
- [x] localize and format dates with moment #9459
- [x] Update translations from Fedora Zanata #9515

### CODE REVIEW

**Comment by martinpitt** [comment]
The test fails as expected. For the future, if you want to test a stacked PR, include the "dependencies" (i. e. the commits from other PRs that you are blocking on), so that the bots already test the combination.

**Comment by martinpitt** [comment]
Both dependency PRs landed, unblocking.

**Comment by Gundersanne** [comment]
I removed the testing of the datepicker, which was removed from #9459,

File: test/verify/check-pages [hunk 1]
```diff
@@ -212,6 +212,97 @@ OnCalendar=daily
...
+        b.enter_page('/system')
+        b.wait_present('#server')
+        b.wait_in_text('#server', 'Reiniciar')
+
+        # Systemd timer localization
+        b.go('/system/services')
+        b.enter_page('/system/services')
+        b.click('#services-filter button[data-pattern=\'\\\.timer\$\']')
+        # HACK: the timers' next run/last trigger (col 3/4) don't always get filled (issue #9439)
+        # b.wait_in_text('tr[data-goto-unit=\'test\.timer\'] td:nth-child(3)', 'morgen um')
```
Comment (martinpitt) [inline test/verify/check-pages]: Would still be good to have the expected value here; although that's not too difficult to get once the bug is fixed.

**Pull request merged**